### ISSUE TITLE
Inconsistent golang type name between protoc-gen-twirp and protoc-gen-go

### ISSUE THREAD

**Issue by tomoemon** [twitchtv/twirp#138]
There is a golang type name inconsistency between a protoc-gen-twirp plugin and a protoc-gen-go plugin.

The protoc-gen-go plugin generates a struct name with camel case (`item_list` message becomes `itemList` struct), but protoc-gen-twirp plugin generates an interface which has parameters with original message name. (refers a `item_list` struct)

# Steps to reproduce

Create a proto file like below. (`sample.proto`)
```proto
syntax = "proto3";
package sample;

service SampleService {
    rpc Get_v1 (SampleReq_v1) returns (SampleRes_v1);
}
message SampleReq_v1 {}
message SampleRes_v1 {}
```

Generate golang files.
```shell
protoc --go_out=./ --twirp_out=./ ./sample.proto
```

Then, protoc-gen-go plugin generates a struct like this,
```go
type SampleReqV1 struct {
...
}
```
but protoc-gen-twirp plugin generates an interface like this.
```go
type SampleService interface {
	GetV1(context.Context, *SampleReq_v1) (*SampleRes_v1, error)
}
```
As there isn't SampleReq_v1 struct anywhere, failed to build.

# Workaround

Change this line
https://github.com/twitchtv/twirp/blob/master/protoc-gen-twirp/generator.go#L1231
to
```go
name += stringutils.CamelCase(def.Descriptor.GetName())
```

**Comment by jtvjorge** [comment]
Twitch are investigating, thank you for the report!

**Comment by dpolansky** [comment]
Hi @tomoemon,

Sorry for the late turnaround on this. This was fixed in #146 and released in v5.5.2.

Thanks for the bug report.
